src/components/calcite-slider/calcite-slider.scss

```diff
@@ -86,9 +86,9 @@ $tick-height: 4px;
       bottom: 0;
     }
     &.hyphen::after {
-      content: "\2014";
-      display: inline-block;
-      width: 1em;
+      content: "\2011";
+      display: inline;
+      margin-left: 0.1875em;
```

Comment:
Is this margin going to be affected by RTL?

Reply:
Great question. It looks like the hyphen appears in the wrong place when it's on an RTL page currently, even when `mirrored` due to the `:after` pseudo-element targeted style. 🤔 

We can fix this if we `getElementDir` inside `hyphenateCollidingRangeHandleLabels`, then add the rtl utility class when the transformed labels overlap:

```tsx
if (rtl) {
  hyphenLabel.classList.add(CSS_UTILITY.rtl)
}
```

```scss
   ...
   &:not(.calcite--rtl).hyphen::after {
     content: "\2011";
     display: inline;
     margin-left: 0.1875em;
   }
   &.calcite--rtl.hyphen::before {
     content: "\2011";
     display: inline;
     margin-left: 0.1875em;
   }
```

which produces this for me on the demo page:

![image](https://user-images.githubusercontent.com/42423180/135354759-1d0f050b-1249-471b-9ffa-b75592cf2b58.png)

Reply:
Can you use a css logical property instead? We don't want to use getElementDir anymore

Reply:
Hm. Without the rtl check + utility class, how can I know whether the `.hyphen` class should have a `:before` or `:after` pseudo-element style?

Reply:
I think if you display it as inline-block it shouldn't need the ::before.

Reply:
I'm noticing even negative numbers appear flipped in slider on an rtl page. Can we do fixes for rtl sliders down the road instead of in this pr?